.gitignore

```diff
@@ -102,3 +102,7 @@ dist
 
 # TernJS port file
 .tern-port
+
+node_modules
+bundle.js
+worker.js
```

Comment:
Why ignore these files?

Don't we need to push them for the solution to work? @Kratika0907